lib/db.c

```diff
@@ -128,6 +129,20 @@ gen_pathname(const char *path, char *buffer, int fno)
   }
 }
 
+static uint32_t target_last_modified = 0;
+
+grn_inline static grn_bool
+grn_db_recover_skip_target(grn_ctx *ctx, grn_obj *obj, uint32_t last_modified)
+{
+  struct stat stat_buf;
+  stat(grn_obj_get_io(ctx, obj)->path, &stat_buf);
```

Comment:
Is needless that check return value of `grn_obj_get_io`?

Reply:
It should be.